src/SSPL-1.0.xml

```diff
@@ -0,0 +1,643 @@
+          conveyed by you (or copies made from those copies), or (b) primarily
+          for and in connection with specific products or compilations that
+          contain the covered work, unless you entered into that arrangement,
+          or that patent license was granted, prior to 28 March 2007.</p>
+                    <p>
+          Nothing in this License shall be construed as excluding or
+          limiting any implied license or other defenses to infringement
+          that may otherwise be available to you under applicable patent law.</p>
+                </item>
+                <item>
+                    <bullet>12.</bullet>
+        No Surrender of Others' Freedom.
+                    <p>
+          If conditions are imposed on you (whether by court order,
+          agreement or otherwise) that contradict the conditions of this
+          License, they do not excuse you from the conditions of this
+          License. If you cannot convey a covered work so as to satisfy
+          simultaneously your obligations under this License and any other
+          pertinent obligations, then as a consequence you may
+                        <br></br>
+          not convey it at all. For example, if you agree to terms
+          that obligate you to collect a royalty for further conveying
+          from those to whom you convey the Program, the only
+          way you could satisfy both those terms and this License
+          would be to refrain entirely from conveying the Program.
+                    </p>
+                </item>
+                <item>
+                    <bullet>13.</bullet>Offering the Program as a Service.
+                    <p>
+If you make the functionality of the Program or a modified version available to third parties as a service, you must make the Service Source Code available via network download to everyone at no charge, under the terms of this License. Making the functionality of the Program or modified version available to third parties as a service includes, without limitation, enabling third parties to interact with the functionality of the Program or modified version remotely through a computer network, offering a service the value of which entirely or primarily derives from the value of the Program or modified version, or offering a service that accomplishes for users the primary purpose of the Program or modified version.</p>
+                    <p>
+&#8220;Service Source Code&#8221; means the Corresponding Source for the Program or the modified version, and the Corresponding Source for all programs that you use to make the Program or modified version available as a service, including, without limitation, management software, user interfaces, application program interfaces, automation software, monitoring software, backup software, storage software and hosting software, all such that a user could run an instance of the service using the Service Source Code you make available.</p>
```

Comment:
I think these `&#8220;` and `&#8221;` tags can just be regular quotation marks instead -- not sure if this will mess up the matching...